docs/02-developers/03-blockchain-essentials/02-overview/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-sidebar_label: Blockchain Overview
+sidebar_label: Rootstock Overview
```

Comment:
Reason for changing the file name is, I found the file is mostly about Rookstock instead of blockchain. So it makes more sense to rename it into `Rookstock Overview` instead of `Blockchain Overview`.